drivers/flash/Kconfig.stm32_ospi

```
@@ -24,3 +24,27 @@ config FLASH_STM32_OSPI
 					$(DT_STM32_OCTOSPI_HAS_DMA)
 	help
 	  Enable OSPI-NOR support on the STM32 family of processors.
+
+if FLASH_STM32_OSPI
+
+if SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U595XX \
+	|| SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U595XX \
+	|| SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U599XX || SOC_STM32U599XX \
+	|| SOC_STM32U599XX || SOC_STM32U599XX || SOC_STM32U599XX || SOC_STM32U599XX \
+	|| SOC_STM32U599XX || SOC_STM32U5A5XX || SOC_STM32U5A5XX || SOC_STM32U5A5XX \
+	|| SOC_STM32U5A5XX || SOC_STM32U5A5XX || SOC_STM32U5A9XX || SOC_STM32U5A9XX \
+	|| SOC_STM32U5A9XX || SOC_STM32U5A9XX
+
+config FLASH_STM32U5_ES0553_2_5_13
```

Nit: a more descriptive name would appreciated, e.g.:
```suggestion
config FLASH_STM32_ERRATUM_U5_STOP2_3_CORRUPT_READ
```